homeassistant/components/knx.py

```diff
@@ -198,7 +206,7 @@ def update(self):
             return False
 
 
-class KNXMultiAddressDevice(KNXGroupAddress):
+class KNXMultiAddressDevice(Entity):
```

Comment:
Why did you decide to duplicate the KNXGroupAddress logic in this class instead of inheriting from it?